schemas/commonUtils/index.ts

```diff
@@ -39,7 +40,8 @@ export const ihclcore: FeatureSchemaDefinition = {
     images,
     membership,
     title,
-    cancellationDropdown
+    cancellationDropdown,
```

Comment:
Why were cancellationDropdown and membershipLogin schema objects added here? Both do not belong to commonUtils. Add them in their respective feature index file.